src/main/resources/attackvectors/ErrorBasedSQLInjectionPayload.properties

```diff
@@ -0,0 +1,20 @@
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=By sending a malicious query into the input fields we should receive a SQL or database syntax error. It will allow us to get information about table names or even query structure.<br/>\
+ <b>How to exploit this level?</b><br/>\
+ Insert this in textbox<br/>\
+ <code>1 OR id=#</code><br/>\
+ As a result, you should get a query that is used by a database management system.
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_2=By sending a malicious query into the input fields we should receive a SQL or database syntax error. It will allow us to get information about table names or even query structure.<br/>\
+ <b>How to exploit this level?</b><br/>\
+ Insert this in textbox<br/>\
+ <code>1 OR id=2</code><br/>\
+ As a result, you should get a query that is used by a database management system.
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_3=By sending a malicious query into the input fields we should receive a SQL or database syntax error. It will allow us to get information about table names or even query structure.<br/>\
+ <b>How to exploit this level?</b><br/>\
+ Insert this in textbox<br/>\
+ <code>#</code><br/>\
+ As a result, you should get a query that is used by a database management system.
+ERROR_BASED_SQL_INJECTION_PAYLOAD_LEVEL_4=By sending a malicious query into the input fields we should receive a SQL or database syntax error. It will allow us to get information about table names or even query structure.<br/>\
+ <b>How to exploit this level?</b><br/>\
+ Insert this in textbox<br/>\
+ <code>#</code><br/>\
+ As a result, you should get a query that is used by a database management system.
```

Comment:
As a result, you should get response with stacktrace showing internal database details. Similar technique hackers can use to get the internal database details and even entire data from the database.